src/main/java/seedu/address/model/order/OrderBeforeDeliveryDateTimePredicate.java

```diff
@@ -0,0 +1,24 @@
+package seedu.address.model.order;
+
+import java.util.function.Predicate;
+
+public class OrderBeforeDeliveryDateTimePredicate implements Predicate<Order> {
+    private final DeliveryDateTime deliveryDateTime;
+
+    public OrderBeforeDeliveryDateTimePredicate(DeliveryDateTime deliveryDateTime) {
+        this.deliveryDateTime = deliveryDateTime;
+    }
+
+    @Override
+    public boolean test(Order order) {
+        return !order.getComplete().isComplete()
```

Comment:
That makes sense. Alternatively I could change the class name to IncompleteOrderBeforeDeliveryDateTimePredicate to make it more precise what it is being used for as I am not sure how simple it would be to work with 2 predicates.